be/src/exprs/vectorized/time_functions.cpp

```diff
@@ -670,25 +670,97 @@ Status TimeFunctions::time_slice_prepare(starrocks_udf::FunctionContext* context
     Slice format_slice = ColumnHelper::get_const_value<TYPE_VARCHAR>(column_format);
     auto period_unit = format_slice.to_string();
 
+    ColumnPtr column_time_base = context->get_constant_column(3);
```

Comment:
must time_slice have the third arg?

Reply:
> must time_slice have the third arg?

Yes, when arguments only 2, we will give default value for 3.